hedera-node/src/main/java/com/hedera/services/store/contracts/repository/MemSizeEstimator.java

```diff
@@ -0,0 +1,9 @@
+package com.hedera.services.store.contracts.repository;
+
+public interface MemSizeEstimator<E> {
+    MemSizeEstimator<byte[]> ByteArrayEstimator = (bytes) -> {
+        return bytes == null ? 0L : (long)(bytes.length + 16);
+    };
+
+    long estimateSize(E var1);
+}
```

Comment:
Do we need a whole class for this? Does it need to require creating an instance?